### PR TITLE
Wip Fix #3717

### LINKED ISSUE #3717: 1.14.5 MP game with ally and my planning mode - OOS errors
Wesnoth v1.14.5 (MP at Official server)
Windows 7 x64

When I played MP games **with ally** (Isar) there was OOS errors in ~50% times. Usually from 1 turn. I am always using planning mode.
oos_error_planning

P. S.

There are problems with planning mode from 1.12 at least. Now we have 1.14.5 and problems become more frequent. What's wrong to simple remove mechanism to show ally your planning - I bet that evil is there. **Simple let player do his planning locally w/o any netcode involved!**

I simply could not play w/o planning mode. And I could not understand how other are playing.
It greatly saves time!

### PR DESCRIPTION
**Pull request by gfgtdf**

### CODE REVIEW

File: src/server/game.hpp [hunk 1]
```diff
@@ -515,6 +517,10 @@ class game
 	/** A wrapper for mersenne twister rng which generates randomness for this game */
 	randomness::mt_rng rng_;
 	int last_choice_request_id_;
+	// first: the oldest wesnoth client version used in this game
+	// second: the newest wesnoth client version used in this game
```
Comment (jyrkive) [inline src/server/game.hpp]: Please don't use `std::pair` for this. Create a custom struct instead.

File: src/server/game.hpp [hunk 2]
```diff
@@ -19,6 +19,7 @@
 #include "player_connection.hpp"
 #include "simple_wml.hpp"
 #include "utils/make_enum.hpp"
+#include "game_version.hpp"
```
Comment (jyrkive) [inline src/server/game.hpp]: Please squash this commit or improve the commit message.

**Comment by CelticMinstrel** [comment]
@gfgtdf Please put the issue number (#3717) in the PR description instead of the title in future.

**Comment by gfgtdf** [comment]
ftr this was merged into 1.14 already, still open becasue im undecided whether i want to put it into master aswell.

**Comment by CelticMinstrel** [comment]
Sure. My statement still stands though.

**Comment by Pentarctagon** [comment]
Closing since the issue this is addressing is already closed.

**Pull request closed without merging**